toolchain/check/handle_name.cpp

```diff
@@ -238,10 +237,56 @@ auto HandleMemberAccessExpr(Context& context, Parse::MemberAccessExprId node_id)
   return true;
 }
 
+auto HandleMemberAccessExpr(Context& context, Parse::MemberAccessExprId node_id)
+    -> bool {
+  return AccessMember(context, node_id);
+}
+
+static auto FollowReference(Context& context, Parse::NodeId node_id) -> bool {
```

Comment:
Per #2006, our term for this operation is "dereference", so I think we should call this something like `DereferencePointer`.